Makefile

```diff
@@ -22,9 +22,9 @@ VENV_EXTRA_ARGS =
 
 $(STATEDIR)/env/pyvenv.cfg : $(_REQUIREMENTS_FILES)
 ifeq ($(PYTHON_VERSION),2)
-	@echo "Using Python 2..."
+	@echo "Using Python 2.7 ..."
 	rm -rf $(STATEDIR)/env
-	virtualenv $(VENV_EXTRA_ARGS) $(STATEDIR)/env
+	virtualenv -p `which python2.7` $(VENV_EXTRA_ARGS) $(STATEDIR)/env
```

Comment:
better bash syntax is `$(which python2.7)` back-tick interpolation is deprecated in bash (back-ticks don't nest, for example, and can be very hard to read/miss depending on fonts etc.) I'm guessing you went with backticks here to get past Makefile variable substitution: just double the dollarsign works `$$(which python2.7)`

Also, I understand this is needed to callout the python version for systems defaulting to `python3` for `virtualenv`. What system is that? That's WTF level broken, in my mind.

You properly handle the python3 spelling `python3 -m venv ` in the other branch, I see.  Apparently, `python2` is not provided on osx, does `brew` not fix that? Or is this part of the "brew defaults to python3" stuff that's been happening?

Reply:
It's simply to ensure python2.7 is used. It's mostly a lapse in my initial coding of this if block, where I was only using python3, so only tested the 'else' branch of logic. But the home brew transition to python3 will definitely cause the issue as well.